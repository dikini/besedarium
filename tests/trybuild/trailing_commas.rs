use besedarium::*;

type TrailingCommaChoice = tchoice!(Http;
    TInteract<Http, EmptyLabel, TClient, Message, TEnd<Http, EmptyLabel>>,
    TInteract<Http, EmptyLabel, TServer, Response, TEnd<Http, EmptyLabel>>,
);

type TrailingCommaPar = tpar!(Http;
    TInteract<Http, EmptyLabel, TClient, Message, TEnd<Http, EmptyLabel>>,
    TInteract<Http, EmptyLabel, TServer, Response, TEnd<Http, EmptyLabel>>,
);

// All trybuild compile-fail tests in this file have been temporarily disabled to stabilize the test base for the TInteract refactor.
