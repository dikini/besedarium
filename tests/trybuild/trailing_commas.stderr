error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/trybuild/trailing_commas.rs:11:3
   |
11 | );
   |   ^ consider adding a `main` function to `$DIR/tests/trybuild/trailing_commas.rs`
